db/schema.rb

```diff
@@ -15,23 +15,21 @@
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
-  create_table "events", force: :cascade do |t|
```

Comment:
This should not be here - its a PR about partials. Furthermore, you should not change your schema unless you have migrations in your PR